internal/storage/postgres/seed/test/demo_data/99_demo_data.sql

```diff
@@ -19,18 +19,17 @@ VALUES
   (8, 'Gewerbegebiet Süd', 'bad', 0.1, 13, 'Schleswiger Straße', 'Sehr viel versiegelter Boden.', 'sandig', 54.768115, 9.435285, ST_SetSRID(ST_MakePoint(54.768115, 9.435285), 4326));   
 ALTER SEQUENCE tree_clusters_id_seq RESTART WITH 9;
 
-INSERT INTO sensors (id, status) VALUES (1, 'online');
-INSERT INTO sensors (id, status) VALUES (2, 'offline');
-INSERT INTO sensors (id, status) VALUES (3, 'unknown');
-INSERT INTO sensors (id, status) VALUES (4, 'online');
-ALTER SEQUENCE sensors_id_seq RESTART WITH 5;
+INSERT INTO sensors (id, status) VALUES ('sensor-1', 'online');
+INSERT INTO sensors (id, status) VALUES ('sensor-2', 'offline');
+INSERT INTO sensors (id, status) VALUES ('sensor-3', 'unknown');
+INSERT INTO sensors (id, status) VALUES ('sensor-4', 'online');
```

Comment:
The demo data in the test should also have the new structure from the sensors

Reply:
Please take a look at the latest update of the file: 
https://github.com/green-ecolution/green-ecolution-backend/blob/feature/edit-database-for-sensors/internal/storage/postgres/seed/99_demo_data.sql